src/current/v25.1/changefeed-for.md

```diff
@@ -71,7 +71,7 @@ Option | Value | Description
 `mvcc_timestamp` | N/A |  Include the [MVCC]({% link {{ page.version.version }}/architecture/storage-layer.md %}#mvcc) timestamp for each emitted row in a changefeed. With the `mvcc_timestamp` option, each emitted row will always contain its MVCC timestamp, even during the changefeed's initial backfill.
-<a name="resolved-option"></a>`resolved` | [`INTERVAL`]({% link {{ page.version.version }}/interval.md %}) | Emits [resolved timestamp]({% link {{ page.version.version }}/changefeed-messages.md %}#resolved-messages) events for the changefeed. Resolved timestamp events do not emit until all ranges in the changefeed have progressed to a specific point in time. <br><br>Set an optional minimal duration between emitting resolved timestamps. Example: `resolved='10s'`. This option will emit a resolved timestamp event **only** if the timestamp has advanced and at least the optional duration has elapsed. If no duration is specified, all resolved timestamps are emitted as the high-water mark advances.<br><br>**Note:** If you set `resolved` lower than `30s`, then you **must** also set the [`min_checkpoint_frequency`](#min-checkpoint-frequency) option to at minimum the same value as `resolved`, because `resolved` messages may be emitted less frequently than `min_checkpoint_frequency`, but cannot be emitted more frequently.<br><br>Refer to [Resolved messages]({% link {{ page.version.version }}/changefeed-messages.md %}#resolved-messages) for more detail.
+<a name="resolved-option"></a>`resolved` | [`INTERVAL`]({% link {{ page.version.version }}/interval.md %}) | Emits [resolved timestamp]({% link {{ page.version.version }}/changefeed-messages.md %}#resolved-messages) events for the changefeed. Resolved timestamp events do not emit until all ranges in the changefeed have progressed to a specific point in time. <br><br>Set a minimum amount of time between the last emitted resolved timestamp and the new resolved timestamp. Example: `resolved='10s'`. This option will emit a resolved timestamp event **only** if the timestamp has advanced and at least the optional duration has elapsed. If no duration is specified, all resolved timestamps are emitted as the high-water mark advances.<br><br>**Note:** If you set `resolved` lower than `30s`, then you **must** also set the [`min_checkpoint_frequency`](#min-checkpoint-frequency) option to at minimum the same value as `resolved`, because resolved messages may be emitted less frequently than `min_checkpoint_frequency`, but cannot be emitted more frequently.<br><br>Refer to [Resolved messages]({% link {{ page.version.version }}/changefeed-messages.md %}#resolved-messages) for more detail.
```

Comment:
Should this also be `[Duration string](https://pkg.go.dev/time#ParseDuration)` like in `create-changefeed.md`? Also, is there any reason this text isn't identical to the one in that file?

Reply:
Updated to duration string, which was missed on the EXP CF FOR page during some earlier update to the resolved option description. 

Please let me know if this should be adjusted, but the main two points that I see are different are: "connected to a sink" and referring to the changefeed as a job. Does `EXPERIMENTAL CHANGEFEED FOR` operate as a job, I thought not, or is that incorrect?